openedxscorm/scormxblock.py

```diff
@@ -202,6 +204,35 @@ def student_view(self, context=None):
         )
         return frag
 
+    @XBlock.handler
+    def scorm_view(self, request, _suffix):
+        """
+        View for serving SCORM content. It receives a request with the path to the SCORM content to serve, generates a pre-signed
+        URL to access the content in the AWS S3 bucket, retrieves the file content and returns it with the appropriate content
+        type.
+
+        Parameters:
+        ----------
+        request : django.http.request.HttpRequest
+            HTTP request object containing the path to the SCORM content to serve.
+        _suffix : str
+            Unused parameter.
+
+        Returns:
+        -------
+        Response object containing the content of the requested file with the appropriate content type.
+        """
+        path = request.url.split('scorm_view/')[-1]
+        path = re.sub(r"(\?.*|:[\d:]*$)", "", path)
+        file_name = os.path.basename(path)
+        signed_url = self.storage.url(os.path.join(self.extract_folder_path, path))
+        file_content = requests.get(signed_url).content
```

Comment:
This is going to blow up the server memory if the object stored at the signed url is a large file. Can we issue a 302 redirect instead?